bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
I imagine that if people wanted to experiment with this library in the terminal, they would use what they are familiar with: `pry --gem`, `bundle console` or `bin/console`.  I think the contents of this file is the same as the contents of bundle console.

I don't know why Bundler adds these file when one asks it to build a gem skeleton.